README.md

```diff
@@ -397,7 +397,24 @@ ANSWERS.addComponent('SearchBar', {
   // Note that WCAG compliance is not guaranteed if a form is not used as the context.
   useForm: 'true',
   // Optional, the input element used for searching and wires up the keyboard interaction
-  inputEl: '.js-yext-query'
+  inputEl: '.js-yext-query',  
+  // Optional, options to pass to the geolocation api.
+  // https://developer.mozilla.org/en-US/docs/Web/API/PositionOptions
+  geolocationOptions: {
+    // Optional, whether to improve accuracy at the cost of response time and/or power consumption, defaults to false.
+    enableHighAccuracy: false,
+    // Optional, the maximum amount of time (in ms) a geolocation call is allowed to take before defaulting, defaults to 1000ms.
+    timeout: 1000,
```

Comment:
I know the spec suggests this timeout, is it reasonable? Experimentally, does this seem like a good default? Most geolocation calls take less than a second?

Reply:
in my experience most take more than one second. Apparently there are a number of other factors that affect speed like whether your device has a gps in it (i.e. is a smartphone)